TestAppTypescript/src/MainScreen.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import SharedStyles from './SharedStyles';
+import { AppState, Alert, Button, View, Platform, ToastAndroid, Text } from 'react-native';
+import AppCenter from 'appcenter';
+import Crashes, { UserConfirmation, ErrorAttachmentLog } from 'appcenter-crashes';
+import Push from 'appcenter-push';
+import AttachmentsProvider from './AttachmentsProvider';
+
+export default class MainScreen extends Component<any, {
+  wrapperSdkVersion: string;
+}>{
+  static navigationOptions = {
+    title: 'TestApp',
+  };
+
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      wrapperSdkVersion: AppCenter.getSdkVersion()
+    }
+  }
+
+  render() {
+    const { navigate } = this.props.navigation;
+
+    return (
+      <View style={SharedStyles.container}>
+        <Text style={SharedStyles.heading}>
+          React Native SDK version {this.state.wrapperSdkVersion}
+        </Text>
+        <Button title="Test Crashes" onPress={() =>
+          navigate('Crashes')
+        } />
+        <Button title="Test Analytics" onPress={() => navigate('Analytics')} />
+        <Button title="Test Push" onPress={() => navigate('Push')} />
+        <Button title="Test Other AppCenter APIs" onPress={() => navigate('AppCenter')} />
+      </View>
+    );
+  }
+}
+
+Push.setListener({
+  onPushNotificationReceived(pushNotification) {
+    let message = pushNotification.message;
+    let title = pushNotification.title;
+    //TODO verify title undefined/null
```

Comment:
Implement the todo since merged with develop that has the fix now.
Please look at TestApp updated code actually as we don't mention android anymore in the message.

Reply:
Not 100% sure on this, can you check upcoming commit?